src/main/java/seedu/tp/flashcard/Flashcard.java

```diff
@@ -9,11 +9,15 @@
     protected String name;
     protected String summary;
     protected List<String> details;
+    protected boolean isReviewed;
+    protected int priorityLevel;
```

Comment:
Consider using enum to define priority level? This way we can force priorityLevel to only take on a restricted set of values and we can also assign meaningful names to the priority levels

Reply:
Sure, do you have any names in mind? I can't think of how to make them more meaningful; first,second,third,fourth,fifth is more troublesome to type than 1,2,3,4,5.

Reply:
I'm thinking maybe we can use HIGH, MEDIUM, LOW, and also include DEFAULT

Reply:
So just 3 + 1 default levels instead of 5? The default currently is set to 'Not indicated' instead of the '***' display.

Reply:
I think it does really matter for us we have 3 or 5 levels? 
Also we can have 3 levels for now and later if we think we need more, we can just add another enum field. 
For the default, we can have the `getPriorityAsString()` method to return the 'Not indicated' string